cubi_tk/isa_tpl/isatab-stem_cell_core_bulk/cookiecutter.json

```diff
@@ -13,8 +13,9 @@
   "library_kit": "mRNA",
   "library_construction_meta": "Library selection,Target insert size",
   "sequencing_meta": "Platform,Instrument model,No. targeted reads,Paired-End",
-  "i_dir_name": "{{cookiecutter.study_title|replace(' ', '_')|replace('/', '-')}}",
+  "study_id": "{{cookiecutter.study_title|replace(' ', '_')|replace('/', '-')}}",
```

Comment:
This doesn't match the scheme of the previous templates (which use s_file_name here ?)
Maybe worth to harmonize this across all template, when things are changed anyway

Reply:
Good point. I removed the study_id variable from this template and stem_cell_core_sc. 

We could also debate if these variables really need to be queried from the user or should be solely defined by things like "Study identifier" (i. e. changed to private variables). What's the use case for custom-named ISA files?